src/main/java/fi/okm/jod/yksilo/entity/koulutusmahdollisuus/Koulutusmahdollisuus.java

```diff
@@ -32,10 +32,12 @@
 import lombok.Getter;
 import org.hibernate.annotations.BatchSize;
 import org.hibernate.annotations.Immutable;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
 @Getter
 @Immutable
+@SQLRestriction("aktiivinen = true")
```

Comment:
I think this won't work, we still need to be able to return the details of the mahdollisuus when queried by id (that is, if a user has a reference to inactive mahdollisuus, they should be able to see it, but the Ehdotus-API should not include inactive ones).